tools/flowc/tests/cpp3/test16.flow

```diff
@@ -26,27 +27,61 @@ Un ::= A, B;
 A(x: int);
 B(y: string);
 
+testCompare(sign: string, structs: [TestStruct], op: (TestStruct, TestStruct) -> bool) {
+	iter(structs, \s1-> iter(structs, \s2 -> 
+		println(toString(s1) + sign + toString(s2) + ": " + b2s(op(s1, s2)))
+	));
+}
+
+printlnAndPass(msg: string, v: ?) -> ? {
+	println(msg + ": " + toString(v));
+	v
+}
```

Comment:
FYI, we have `pprint` function https://github.com/area9innovation/flow9/blob/c85c358e81e13d3f37d4c73be8b0164770a50ac7/lib/runtime.flow#L477

Reply:
Thanks, printlnAndPass is replaced with pprint in tests